frontend/pages/club/[club]/renew.tsx

```diff
@@ -222,7 +222,11 @@ const RenewPage = (props: RenewPageProps): ReactElement => {
   }
 
   if (SITE_ID === 'fyh') {
-    return <ResourceCreationPage {...props} />
+    return (
+      <ResourceCreationPage
+        {...{ ...props, student_types: props.studentTypes }}
```

Comment:
The outer later of `{...  }` is unnecessary.

Also, I would refactor the resource creation page to accept studentTypes instead of changing this page to pass in `student_types`. Camel case is more inline with Javascript code style.

Reply:
Just noticed that `ClubForm` from ClubEditPage.tsx also does this. Seems like it takes parameters from `doBulkLookup` which kind of enforces snake case naming. Should we fix this for both components?